src/cl_sii/rcv/data_models.py

```diff
@@ -562,6 +563,17 @@ def validate_contribuyente_razon_social(cls, v: object) -> object:
             cl_sii.dte.data_models.validate_contribuyente_razon_social(v)
         return v
 
+    @pydantic.field_validator('tasa_otro_impuesto')
+    @classmethod
+    def parse_tasa_otro_impuesto(cls, v: object) -> object:
```

Comment:
After inspecting the CSV file that caused the issue, I found that the cause was that a `"9.26"` value, which is parsed as a `float`, when converted later to a `Decimal` ends up as `Decimal('9.2599999999999997868371792719699442386627197265625')`. Maybe we should just skip the `float` conversion and change the definition of the `tasa_otro_impuesto` field from `Optional[float]` to `Optional[Decimal]`. Then, this `field_validator` shouldn't be necessary anymore.